From 2ca3582d493a7166e8344ded744eced5c0c04664 Mon Sep 17 00:00:00 2001
From: Derek Higgins <derekh@redhat.com>
Date: Tue, 15 Jul 2014 23:16:19 +0000
Subject: [PATCH] Unwrap setting of SECRET_KEY

This will ensure the sed setting SECRET_KEY in the spec wont
break the python syntax.
---
 openstack_dashboard/local/local_settings.py.example | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/openstack_dashboard/local/local_settings.py.example b/openstack_dashboard/local/local_settings.py.example
index d4b9583..736835e 100644
--- a/openstack_dashboard/local/local_settings.py.example
+++ b/openstack_dashboard/local/local_settings.py.example
@@ -94,8 +94,7 @@ HORIZON_CONFIG = {
 # for all of them.
 from horizon.utils import secret_key
 LOCAL_PATH = '/tmp'
-SECRET_KEY = secret_key.generate_or_read_from_file(
-    os.path.join(LOCAL_PATH, '.secret_key_store'))
+SECRET_KEY = secret_key.generate_or_read_from_file(os.path.join(LOCAL_PATH, '.secret_key_store'))
 
 # We recommend you use memcached for development; otherwise after every reload
 # of the django development server, you will have to login again. To use
-- 
1.9.3

